config/nginx-default.conf

```diff
@@ -0,0 +1,28 @@
+server {
+    server_name localhost;
+    listen 80;
+    root /srv/annotations/web;
```

Comment:
this is a bit weird as the nginx container doesn't have this folder, and we don't mount it. But it's passed to php-fpm along with `app_dev.php` to find the entry point of the application